ViewModel/AggregatedStageDamage/StageDamageCurve.cs

```diff
@@ -34,8 +34,11 @@ public StageDamageCurve(XElement curveElement)
             int selectedImpArea = int.Parse(curveElement.Attribute(SELECTED_IMPACT_AREA_TAG).Value);
             string selectedDamCat = curveElement.Attribute(SELECTED_DAM_CAT_TAG).Value;
             AssetCategory = curveElement.Attribute(ASSET_CATEGORY).Value;
-            //TODO: get rid of hard-coded string
             XElement functionElem = curveElement.Element("CurveComponentVM");
+            if (functionElem == null)
+            {
+                functionElem = curveElement.Element("ComputeComponentVM");
```

Comment:
nicely done. it might be helpful to have a little comment here about why this logic exists.